pkgs/development/tools/mold/default.nix

```diff
@@ -37,11 +37,10 @@ stdenv.mkDerivation rec {
   };
 
   meta = with lib; {
-    description = "A high performance drop-in replacement for existing unix linkers";
+    description = "A high performance drop-in replacement for existing Unix linkers";
     homepage = "https://github.com/rui314/mold";
-    license = lib.licenses.agpl3Plus;
-    maintainers = with maintainers; [ nitsky ];
-    platforms = platforms.unix;
+    license = licenses.agpl3Plus;
+    maintainers = with maintainers; [ azahi nitsky ];
     # error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.14 or newer
```

Comment:
This can be addressed with

```
  NIX_CFLAGS_COMPILE = lib.optional stdenv.cc.isClang "-faligned-allocation";
```

but `autoPatchelfHook` also needs to be removed to allow it to build on darwin.